.github/workflows/rust.yml

```diff
@@ -247,3 +247,13 @@ jobs:
       with:
         name: cargo-doc
         path: target/doc
+  deploy:
+    needs: bench
```

Comment:
I think we can publish after `docker tests`? Why do it after `bench`?

Reply:
I usually set delivery as a last stage in CI to prevent publishing of artifacts that may fail, like here on benchmarks.

Reply:
But the bench may not be the last to finish, these branches of checks may be working in parallel:
- check -> test -> bench
- check -> test-docker

Reply:
Thanks Egor - I will check is there a possibility to define stages instead of job deps

Reply:
Looks like not - so I will depend on docker tests as you suggested.

Reply:
Actually I think we can do better maybe let's transform the path to the following:
check -> test -> test-docker -> deploy-docker-image
Then it will be more logical if docker image is published after all those tests.